test/env/array/array-env.test.js

```diff
@@ -10,15 +10,15 @@ const devFile = path.join(__dirname, './bin/dev.js');
 const prodFile = path.join(__dirname, './bin/prod.js');
 
 describe('env array', () => {
-    it('is able to set two different environments for an array configuration', () => {
+    it('is able to set two different environments for an array configuration', async () => {
         const { stderr, stdout, exitCode } = run(__dirname);
 
         expect(exitCode).toBe(0);
         expect(stderr).toBeFalsy();
         expect(stdout).toBeTruthy();
 
-        const devScript = spawnSync('node', [devFile]);
-        const prodScript = spawnSync('node', [prodFile]);
+        const devScript = await spawnSync('node', [devFile]);
```

Comment:
why need await when spawnSync is synchronous by nature?

Reply:
I have reverted it lets see if CI is happy with it 😄